build/bazelutil/nogo_config.json

```diff
@@ -1678,6 +1678,11 @@
             "github.com/cockroachdb/cockroach/.*$": "first-party code"
         }
     },
+    "U1000": {
+        "only_files": {
+            "github.com/cockroachdb/cockroach/.*$": "first-party code"
```

Comment:
fwiw, the reason we moved all of our own code under `pkg` was so that that prefix would denote first-party code, and thus leave room at the repo root for other directories like c-deps, vendor, or whatever that may not be first party. For now, bazelized builds don't look at those directories so this is slightly academic.